docs/vendors.md

```diff
@@ -36,6 +36,17 @@ Service components:
 
 Each _model component_ can be paired with the appropriate _service provider component_ to pair an _AI Agent_. This _agent_ communicates with the model running on a given hardware (e.g. `Claude Messages API` with `Claude HTTP Service Provider` to run `Claude` model in a cloud).
 
+The table below shows the compatibility between the model components and service provider components.
+
+| Model component | Service provider component | Notes |
+| --- | --- | --- |
+| Claude Messages API | Claude HTTP | Supports `Claude 2.1` and all of `Claude 3` versions. The HTTP service provider should point to the appropriate URL for use with the messages API. A `ANTHROPIC_API_KEY` environment variable must be available. |
+| Claude Messages API | Claude Amazon Bedrock | Supports `Claude 2.1` and all of `Claude 3` versions available in the selected region. Needs to be used with a valid AWS SDK credentials sourced in the environment. |
+| Claude Text Completions | Claude HTTP | Supports `Claude 2.1` version. The HTTP service provider should point to the appropriate URL for use with the Text Completions API. A `ANTHROPIC_API_KEY` environment variable must be available. |
+| Claude Text Completions | Claude Amazon Bedrock | Supports `Claude 2.1` version. Needs to be used with a valid AWS SDK credentials sourced in the environment. |
+| Ollama Chat | Ollama HTTP | Supports the `Ollama` model. Use with the `chat` functionality of Ollama by setting the appropriate URL. |
+| Ollama Generate | Ollama HTTP | Supports the `Ollama` model. Use with the `generate` functionality of Ollama by setting the appropriate URL.|
```

Comment:
Is it `Ollama model`? Shouldn't you point to _mistral_ or _llama3_ or anything like that?

Reply:
I can add that a model needs to be selected in the notes. Ollama is a model component, even tho it supports multiple models.